packages/target-tauri/translationfn/src/lib.rs

```diff
@@ -0,0 +1,370 @@
+#![forbid(clippy::indexing_slicing)]
+#![warn(
+    unused,
+    clippy::correctness,
+    clippy::all,
+    clippy::wildcard_imports,
+    clippy::needless_borrow,
+    clippy::cast_lossless,
+    clippy::unused_async,
+    clippy::explicit_iter_loop,
+    clippy::explicit_into_iter_loop,
+    clippy::cloned_instead_of_copied
+)]
```

Comment:
Is this here by mistake? If not, IMU this turns `correctness` from error into a warning, right? I don't think we want that.